jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -538,8 +538,8 @@ def _image_viewer_update(self, viewer, x, y, mouseevent=True):
             row1a_text = ""
             row1a_title = ""
         elif getattr(image, 'meta', {}).get('_importer', '') == 'CatalogImporter':
-            row1a_text = ''
-            row1a_title = ''
+            row1a_text = str(image.data['ID'][cur_i])
+            row1a_title = 'Source ID'
```

Comment:
can/should this use the original name of the column?  Maybe since this is only used for mouseover and not any linking, we could just store the column name in the metadata rather than having to hardcode a label (and just avoid duplications if its also listed in other columns)?